molgenis/molgenis-emx2
# feat: added support for named tables, columns, and subselections in aggregation dashboard

What are the main changes you did:
- The aggregation dashboard was renamed to `aggregates` so that it is easier to type
- Expanded the underlying function `getChartData` to retrieve data from any table
- Added support for specifying the subselection attribute in the graphql groupBy queries (`id`, `name`, `displayName`, etc.)
- Fixed typescript errors
- Removed collections filter, header, and header image

how to test:
- Navigate to the preview and go to the aggregates dashboard at: https://preview-emx2-pr-4055.dev.molgenis.org/aggregateTest/aggregates/#/
- Click filters and note the changes to the updated chart. Compare the results to the table: https://data-catalogue.molgeniscloud.org/Aggregates/tables/#/FORCE_NENPatients

todo:
- [x] use display name rather than splitting center names (see PR #3924)
- [x] Fix issue with research center in filters. I don't think `id` is passed down to the filter. I suspect `name ` is still used.
